ice.c

```diff
@@ -3617,7 +3617,21 @@ static gint rtcp_transport_wide_cc_stats_comparator(gconstpointer item1, gconstp
 static gboolean janus_ice_outgoing_transport_wide_cc_feedback(gpointer user_data) {
 	janus_ice_handle *handle = (janus_ice_handle *)user_data;
 	janus_ice_peerconnection *pc = handle->pc;
-	janus_ice_peerconnection_medium *medium = pc ? g_hash_table_lookup(pc->media_bytype, GINT_TO_POINTER(JANUS_MEDIA_VIDEO)) : NULL;
+
+	janus_ice_peerconnection_medium *medium;
```

Comment:
`medium` should be initialized to `NULL`, since there's no guarantee the check you added will find something (while the `g_hash_table_lookup` we had can return `NULL` in that case).